pennylane_forest/qpu.py

```diff
@@ -139,11 +141,7 @@ def expval(self, observable, shot_range=None, bin_size=None):
                 pauli_obs = sZ(wire)
 
             # Multi-qubit observable
-            elif (
-                len(device_wires) > 1
-                and isinstance(observable, Tensor)
-                and not self.parametric_compilation
```

Comment:
Why was this changed? This seems to allow parametric compilation with op estimation, do they work well together in this plugin?

Reply:
We already check that `self.parametric_compilation` is False at the beginning of this method. This check here did nothing.

Reply:
I don't think they work well together, so we still disable operator estimation when `parametric_compilation = True`

Reply:
Good catch! Thanks for explaining.